src/cli/cli.cpp

```diff
@@ -2681,13 +2689,49 @@ template <> otError Interpreter::Process<Cmd("csl")>(Arg aArgs[])
         error = ProcessSet(aArgs + 1, otLinkSetCslTimeout);
     }
     else
```

Comment:
We seem to leave a hanging `else` here? (which will then combine with the `if` below)?